banking-bench/src/main.rs

```diff
@@ -121,6 +144,34 @@ fn make_accounts_txs(
         .collect()
 }
 
+// In siumulate mint, 99% transactions in a batch are mint transaction (eg., have conflicting account and higher
+// priority) and 1% regualr transactions (eg., non-conflict and low priority)
+fn is_simulated_mint_transaction(
+    simulate_mint: bool,
+    index: usize,
+    packets_per_batch: usize,
+    mint_txs_percentage: usize,
+) -> bool {
+    !simulate_mint || (index % packets_per_batch <= packets_per_batch * mint_txs_percentage / 100)
```

Comment:
if `simulate_mint` is false this returns true.

```suggestion
    simulate_mint && (index % packets_per_batch <= packets_per_batch * mint_txs_percentage / 100)
```

Reply:
dang, not sure your test cmd, but could it be the cause of weird 0% test result?